tests/bench/harness.py

```
@@ -0,0 +1,683 @@
+"""
```

Ruff (0.12.2)

The file is executable but no shebang is present

(EXE002)